ROOTFS/etc/sage-utils/dns/nfs/bin/ro-root.sh

```diff
@@ -1,50 +1,4 @@
 #!/bin/sh
-
```

Comment:
I am fine removing this comment but I think we should have some form of comment or information in the git commit comment about where this file was sourced from.